picard/ui/metadatabox/__init__.py

```diff
@@ -142,9 +142,21 @@ class MetadataBox(QtWidgets.QTableWidget):
     MIMETYPE_TSV = 'text/tab-separated-values'
     MIMETYPE_TEXT = 'text/plain'
     mimedatas = {
-        MIMETYPE_PICARD_TAGS: 'to_json',
-        MIMETYPE_TSV: 'to_tsv',
-        MIMETYPE_TEXT: 'to_tsv',
+        MIMETYPE_PICARD_TAGS:
+        {
+            'copy_to_mime': lambda data: data.to_json(),
```

Comment:
Using a namedtuple could be better here, something like:

```python

MimeConverters = namedtuple('MimeConverters', ('copy_to_mime', 'mime_data_paste'))

....
    mimedatas = {
        MIMETYPE_PICARD_TAGS: MimeConverters(
            copy_func=lambda data: data.to_json(),
	    paste_func=lambda metadatabox, mimedata: metadatabox._paste_from_json(mimedata),
        ),
....


           for mimetype, converters in self.mimedatas.items():
                if converter.copy_func:

....

        for mimetype, converters in self.mimedatas.items():
            if mimedata.hasFormat(mimetype):
                # Call the paste function to get the data
                if converters.paste_func:
                    objs = converters.paste_func(self, mimedata)
 ```

Reply:
Yep, I was thinking it would end up like that! Wanted to make sure I wasn't going down a rabbit hole first

Reply:
I think this is going in the right direction, update this PR with suggested changes if they fit you so we can merge to main PR and see what others say about it.